lib/packwerk/commands/uses_parse_run.rb

```diff
@@ -0,0 +1,91 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "optparse"
+
+module Packwerk
+  module Commands
+    module UsesParseRun
+      extend T::Sig
+      extend T::Helpers
+
+      requires_ancestor { BaseCommand }
+
+      sig do
+        params(
+          args: T::Array[String],
+          configuration: Configuration,
+          out: T.any(StringIO, IO),
+          err_out: T.any(StringIO, IO),
+          progress_formatter: Formatters::ProgressFormatter,
+          offenses_formatter: OffensesFormatter,
+        ).void
+      end
+      def initialize(args, configuration:, out:, err_out:, progress_formatter:, offenses_formatter:)
+        super
+        @_parsed_options = T.let(nil, T.nilable(T::Hash[Symbol, T.untyped]))
```

Comment:
Ah, I used an underscore because it's a memoization ivar for the `parsed_options` method, and Sorbet seemed to require this `T.let` in the initializer. This might not be true though, I'll check.

Reply:
4ccb564944e7c1a3ac0cddb413cd9969dfdc4bf1